src/Microsoft.DotNet.Build.Tasks.Packaging/src/NuGetPack.cs

```diff
@@ -243,11 +301,60 @@ public void Pack(string nuspecPath, Func<string, string> nuspecPropertyProvider,
             }
             catch (Exception e)
             {
-                string packageType = packSymbols ? "symbol" : "lib";
+                string packageType = "lib";
+                if (creatingSymbolsPackage)
+                {
+                    packageType = "symbol";
+                }
+                else if (packSymbols)
+                {
+                    packageType = "packed";
+                }
                 Log.LogError($"Error when creating nuget {packageType} package from {nuspecPath}. {e}");
             }
         }
 
+        private Manifest TransformManifestToPackedPackageManifest(Manifest manifest)
```

Comment:
Why wouldn't we want both?  We'd still want symbols to be available for scenarios where the transport package isn't being consumed.  I think we need all three because CoreClr is still producing stand-alone packages, so for them we want standard, symbols, and transport packages.